chance/p5js/chance-praegnanz/index.md

```diff
@@ -3,9 +3,9 @@ layout: p5js
 author: josues
 author-url: https://incom.org/profil/7029
 title: Law of praegnanz and chance
-tags: P5.js, chance, proximity, Law of praegnanz
+tags: [P5.js, chance, proximity, Law of praegnanz]
```

Comment:
Nearly there the laws are written like titles. For the " Law of Prägnanz" I also used the "umlaute".